share/ext/astcenc/build_astcenc.sh

```diff
@@ -94,10 +177,10 @@ case $PLATFORM in
         LIB_NAME=libastcenc-neon-static.a
         ;;
     x86_64-linux)
-        LIB_NAME=libastcenc-native-static.a
+        LIB_NAME=libastcenc-avx2-static.a
```

Comment:
Is the avx2 version fallback compatible?
As not all devices have avx2, it may be problematic otherwise?